src/Diagnostics/CodeAnalysis/Setup/CodeAnalysisDiagnosticsSetup.csproj

```diff
@@ -60,14 +60,20 @@
     <ProjectReference Include="..\Core\CodeAnalysisDiagnosticAnalyzers.csproj">
       <Project>{d8762a0a-3832-47be-bcf6-8b1060be6b28}</Project>
       <Name>CodeAnalysisDiagnosticAnalyzers</Name>
+      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup</IncludeOutputGroupsInVSIX>
+      <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup</IncludeOutputGroupsInVSIXLocalOnly>
```

Comment:
It means that the local deploy of the extension to the hive should include those files. i.e. "Please copy the PDBs along when debugging, but don't include that in the regular package"